sif_parser/sif_open.py

```diff
@@ -150,3 +151,77 @@ def xr_open(sif_file, ignore_corrupt=False, lazy=None):
     return xr.DataArray(data, dims=['Time', 'height', 'width'],
                         coords=coords, attrs=new_info)
 
+def spool_open(spool_dir, ignore_missing=False):
```

Comment:
Could you rename it to `np_spool_open`? We may eventually want `xr_spool_open` as well.
Also, could you add `lazy=None` as a place holder? We may eventually want a support of lazy load.